Alignment/APEEstimation/python/MisalignmentScenarios_cff.py

```diff
@@ -1238,3 +1238,30 @@
         ),
     ),
 )
+
+MisalignmentScenario_PhaseI_PseudoAsymptotic = cms.PSet(
```

Comment:
@ghellwig sorry, I realized I didn't answer your question: I mean add a reference in the code itself. 
Actually why is there a `Alignment/APEEstimation/python/MisalignmentScenarios_cff.py` file at all? 
Can we try to aggregate everything here: https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Alignment/TrackerAlignment/python/Scenarios_cff.py where previous scenarios were documented?